spec/board_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'spec_helper'
+require 'board'
+
+describe Board do
+  # let(:board)  { Board.new }
+  let(:region) { double :region }
+  let(:board) { Board.new}
```

Comment:
Just a sidenote - unless you specify it yourself, the `subject` keyword corresponds to `described_class.new`